packages/data-explorer/src/components/dataView/TableHeader.vue

```diff
@@ -2,6 +2,7 @@
   <thead>
   <tr>
     <th scope="col" v-if="isShop"></th>
+    <th scope="col" v-if="!isShop"></th>
```

Comment:
I don't see the reason to do this.. Is this useful?

Reply:
the column is there for two different reasons ( shop stuff or actions stuff), i agree the code looks silly now , i wanted to make it explicit so we can refactor this once we are clear on the UI/UX design